package.json

```diff
@@ -26,14 +26,17 @@
     "printWidth": 80
   },
   "dependencies": {
-    "@dcl/schemas": "^4.6.0",
+    "@dcl/schemas": "^6.12.0",
     "@well-known-components/env-config-provider": "^1.1.1",
-    "@well-known-components/http-server": "^1.0.0",
-    "@well-known-components/interfaces": "^1.1.0",
-    "@well-known-components/logger": "^1.1.0",
+    "@well-known-components/http-requests-logger-component": "^0.1.0-20230308121414.commit-d9bf633",
```

Comment:
is this a development version?

Reply:
Great catch! I've just replaced it with the latest version that also includes an additional feature.